client/my-sites/stats/stats-module/connected-list.jsx

```diff
@@ -119,12 +178,14 @@ class StatsConnectedModule extends Component {
 		);
 
 		const summaryLink = this.getHref();
-		const displaySummaryLink = data && data.length >= 10;
+		const displaySummaryLink = listData && listData.length >= 10;
 		const isAllTime = this.isAllTimeList();
 
 		return (
 			<div>
 				{ siteId && statType && <QuerySiteStats statType={ statType } siteId={ siteId } query={ query } /> }
+				{ siteId && statType && showPriorPeriodToggle && showingPriorPeriod &&
```

Comment:
Have you considered, showing the query even if the `showingPriorPeriod` is not true? It could act like a "preload" and if we're concerned about having too many requests at the same time, we may delay those (when the currentQueryIsRunning or with a timeout).